### DISCUSSION TITLE
error: exportArchive: "Runner.app" requires a provisioning profile

### DISCUSSION THREAD

**Discussion by beky02** [codemagic-ci-cd/codemagic-docs#2489] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hi there,

am trying to build a Flutter white label app and followed this https://docs.codemagic.io/yaml-code-signing/alternative-code-signing-methods/ to configure Automatic code signing

am having a problem with the flutter build ipa command to build the iOS archive. It completes with this log:

<img width="845" alt="Screenshot 2024-04-23 at 11 40 59 AM" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/37780908/fa6cee90-1f5d-4ca2-8e3a-3d865cca85d9">

### Expected results

build ios app and publish to testflight 

### Actual results

error: exportArchive: "Runner.app" requires a provisioning profile

**Comment by himesh-cm** [comment]
Can you please share the build ID with me? You can find it under your build details page's 'ID' section.  ﻿Navigate to the 'Builds' section on the left-hand menu, select the build you want to view, and click the copy icon.

**Comment by dtrdic** [comment, marked as answer]
Note: issue was resolved over Discord.There was a syntax error in the build command that was causing this issue
